src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor_Constants.cs

```diff
@@ -13,7 +13,7 @@ private void AddConstantValue(ITypeSymbol type, object constantValue, bool prefe
             {
                 AddNonNullConstantValue(type, constantValue, preferNumericValueOrExpandedFlagsForEnum);
             }
-            else if (type.IsReferenceType || type.TypeKind == TypeKind.Pointer || type.OriginalDefinition.SpecialType == SpecialType.System_Nullable_T)
+            else if (type.IsReferenceType || type.TypeKind == TypeKind.Pointer || ITypeSymbolHelpers.IsNullableType(type))
```

Comment:
:bulb: `type.IsNullableType()`

Reply:
> type.IsNullableType()

This will not work. The method is not member or an extension method.